en/companion_computer/pixhawk_rpi.md

```diff
@@ -0,0 +1,289 @@
+# Raspberry Pi Companion with Pixhawk
+
+This topic describes how to setup a Raspberry Pi ("RPi") companion companion running ROS2 on Linux Ubuntu OS, connecting to a [Pixhawk](../flight_controller/autopilot_pixhawk_standard.html) flight controller.
+
+There are different models of Pixhawk and RPi.
+The article illustrates the setup using [Pixhawk Standard](../flight_controller/autopilot_pixhawk_standard.html) controllers, but the PX4 configuration for other controllers is likely to be similar (or use the same concepts).
+Similarly, there are several models of RaPi.
+This provides the main concepts, and is extensible to most configurations.
+
+In addition, there are a number of different ways that RPi can be connected with a Pixhawk:
+
+1. Serial connection. This uses a serial port, typically `TELEM2`, or the `USB` port.
+1. Ethernet. Pixhawk controllers based on FMUv5x, 6x and later may have an inbuilt Ethernet port.
+   See [PX4 Ethernet > Supported Controllers](../advanced_config/ethernet_setup.md#supported-flight-controllers).
+
+This article covers in the first instance connecting the `TELEM2` port with the RPi's TX/RX pins.
+This is a simple configuration because you do not need to use an FTDI Chip USB-to-serial adapter board.
+Note that we hope to extend the document for other connection methods in future.
+
+## Serial Connection
```

Comment:
What I'd like to do is extend this for Ethernet setup later maybe. So I was thinking of having all the rest under Serial connection. Not sure yet. I'll proceed next week.